apps/crn-frontend/src/analytics/engagement/Engagement.tsx

```diff
@@ -39,6 +39,8 @@ const Engagement = () => {
     timeRange,
   });
 
+  const performance = useEngagementPerformance({ timeRange: '30d' });
+
   const { numberOfPages, renderPageHref } = usePagination(total, pageSize);
 
   const exportResults = () =>
```

Comment:
Hey, I think we should also update export results since it should show the performance https://docs.google.com/spreadsheets/d/1ito19ZHryOSu0JxE1YQMltYkI7S_3S8PmwZgbHJb72Y/edit?gid=1516875906#gid=1516875906

<img width="2290" alt="Screenshot 2024-09-06 at 05 49 28" src="https://github.com/user-attachments/assets/8f711aa9-fb1e-4fa2-9c0e-5ac3283b44ed">